src/Controller/UserController.php

```diff
@@ -21,6 +21,9 @@ class UserController extends AbstractController
      * @Route("/profile/{id}", name="profile", priority=10)
      */
     public function profile(EntityManagerInterface $em, Request $request, User $user){
+
+        $this->denyAccessUnlessGranted('USER_PROFILE', $user);
+
```

Comment:
Ned pas injecter `User $user`, modifier la route en supprimant la variable `/{id}`. Et par conséquent passer directement `$this->getUser()`

Reply:
Modifié!